src/components/cards/EncountersCard.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { useState, useMemo } from 'react';
-import { useIntl } from 'react-intl';
+import { useIntl, FormattedMessage } from 'react-intl';
```

Comment:
The changes in this code are a partial fix to the related DEX-770. Ticket. These changes are incomplete. 

I could put these changes in place before approval of this PR if that makes logistics easier.

Reply:
* DEX-771, not DEX-770. This is DEX-770. Omg.